src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/RunReportPageTests.kt

```diff
@@ -102,4 +106,47 @@ class RunReportPageTests : IntegrationTest()
                 ReportWithDate("minimal", repo.getLatestVersion("minimal").date)
         )
     }
+
+    @Test
+    fun `can get running reports details`()
+    {
+        val instant = Instant.now()
+        val fakeReportRunLog = ReportRunLog(
+                "test@example.com",
+                instant,
+                "q123",
+                "{‘0’: {‘source’: ‘support’, ‘annexe’: ‘annexe val’}}",
```

Comment:
Not important for tests but just to clarify earlier comment, this field will actually be received in this form:
```
{"source": "production", "extra": "staging"}
```
In order words, a JSON object mapping a datasource name to the instance of the relevant database